src/shell/bash.rs

```diff
@@ -14,10 +14,20 @@ pub fn spawn_shell(info: &ShellSpawnInfo) -> Result<()> {
     write!(
         temp_rc_file,
         r#"
-if [ -f "$HOME/.bashrc" ] ; then
-    source "$HOME/.bashrc"
-elif [ -f "/etc/skel/.bashrc" ] ; then
-    source /etc/skel/.bashrc
+# OS X creates a login shell instead of a normal shell, which means that
+# a different set of files contain the bash configuration.
+if [[ "$OSTYPE" == "darwin"* ]] ; then
```

Comment:
Missed this during the initial review yesterday... [The logic for profile loading seems to be a bit more complex](https://github.com/sbstp/kubie/issues/13#issuecomment-611958662)...